app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    source = command.split()
```

Comment:
Consider trimming the command string before splitting to handle any leading or trailing spaces. This can prevent unexpected behavior if the command string is not formatted correctly.